helpers/labgrid-raw-interface

```diff
@@ -32,7 +32,7 @@ def get_denylist():
     return denylist
 
 
-def main(program, ifname, count):
+def main(program, ifname, count, timeout=0):
```

Comment:
Don't specify defaults in two locations (here and in argparse as None). Just drop the default here, as this function is always called with the full arguments.